packages/client/components/ReflectionGroup/ReflectionGroup.tsx

```diff
@@ -116,8 +116,10 @@ const ReflectionGroup = (props: Props) => {
   })
   const onClick = () => {
     if (isEditing) return
-    const wasDrag = staticReflections.some((reflection) => reflection.isDropping)
-    if (wasDrag) return
+    const isNotDraggable = staticReflections.some(
+      (reflection) => reflection.isDropping || reflection.inSpotlight
```

Comment:
If you have a stack of reflections, click the `showSpotlight` icon and close the spotlight modal, the reflection stack modal will then open up when we don't want it to. Using `stopPropagation` when clicking the `showSpotlight` icon doesn't prevent it from happening so it looks like we need to prevent it in the `onClick` here. 

To do this, I've added an `inSpotlight` field to the Relay client. This seems like overkill & is confusing as we also need to pass an `inSpotlight` prop to the `draggableReflection` in the modal so the `showSpotlight` icon doesn't show (Relay doesn't update in time to work in both). I'll work on finding a better solution but any suggestions would be great!